src/features/Dashboard/InstructorAssignSection/ClassCard.jsx

```diff
@@ -17,10 +19,28 @@ const ClassCard = ({ data }) => {
     history.push(addQueryParam(`/manage-instructors/${encodeURIComponent(data?.masterCourseId)}/${encodeURIComponent(data?.classId)}?previous=dashboard`));
   };
 
+  const classElement = 'className';
```

Comment:
Why do you use a variable if it is not used in another part of the file and it is just a string?